components/nimbus/ios/Nimbus/NimbusCreate.swift

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+private let EXPERIMENT_BUCKET_NAME = "main"
+private let EXPERIMENT_COLLECTION_NAME = "nimbus-mobile-experiments"
```

Comment:
Swift lint doesn't like these names.

Reply:
I would recommend following the Swift naming convention for constants (explained well here: https://github.com/raywenderlich/swift-style-guide#constants)

Reply:
More of a note-to-self than a cry-for-help 😃

Reply:
I get bitten by this one a lot, I too like the all caps constants :)

Reply:
I went with your suggestion of using wenderlich styling.